src/mito2/src/worker/handle_manifest.rs

```diff
@@ -203,9 +204,16 @@ impl<S> RegionWorkerLoop<S> {
 
         let RegionEditRequest {
             region_id: _,
-            edit,
+            mut edit,
             tx: sender,
         } = request;
+        let file_sequence = region.version().flushed_sequence + 1;
+        edit.flushed_sequence = Some(file_sequence);
+
+        // For every file added through region edit, we should fill the file sequence
+        for file in &mut edit.files_to_add {
+            file.sequence = NonZeroU64::new(file_sequence);
+        }
```

Comment:
Is it possible that the flushed_sequence is greater than the committed sequence if we bump it too many times?